RedHat-UX/red-hat-design-system
# docs(tile): fix light theme selected state image

## What I did

Closes #1682 

The image for the selected state of selectable tile (light theme) was showing two radio button tiles. I updated this image to show one radio button tile and one checkbox tile.

## Testing Instructions

Check selectable tile's [selected state](https://deploy-preview-1683--red-hat-design-system.netlify.app/elements/tile/style/#selected) in the deploy preview.

## Notes to Reviewers
